tests/fixtures/birth_names_dashboard.py

```diff
@@ -127,7 +127,7 @@ def _get_dataframe(database: Database) -> DataFrame:
     data = _get_birth_names_data()
     df = pd.DataFrame.from_dict(data)
     if database.backend == "presto":
-        df.ds = df.ds.dt.strftime("%Y-%m-%d %H:%M%:%S")
+        df.ds = df.ds.dt.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
Bycatch: fix simulation data for Presto timestamp, which may cause error when the `ds` column is used in `cast(ds as TIME)`